components/eam/bld/namelist_files/namelist_definition.xml

```diff
@@ -3644,6 +3644,60 @@ Switch for gas chemistry tracers 2D budget diagnostic output
 Default: .false.
 </entry>
 
+<entry id="history_gaschmbudget_2D_levels" type="logical"  category="diagnostics"
+       group="phys_ctl_nl" valid_values="" >
+Switch for gas chemistry tracers 2D budget diagnostic output within certain levels
+Default: .false.
+</entry>
+
+<entry id="gaschmbudget_2D_L1_s" type="integer" category="diagnostics"
+       group="phys_ctl_nl" valid_values="" >
+Start layer of L1 for gas chemistry tracers 2D budget diagnostic output
```

Comment:
Shall we change "Start layer" to "Start layer index" to be more precise? If so, should be changed for other similar variable descriptions too.

Reply:
I prefer to use the "Start layer". "Index" sounds more confusing to me.